### PR TITLE
Add dms to benchmark

### PR DESCRIPTION
**Pull request by harikt**
So dms is good for a merge though currently the caching is not taken care. 

Probably @TimeToogo is the best person to look into this and see if there is more refactoring to be done on the PR.

### CODE REVIEW

File: provider/dms/Domain/Entities/Comment.php [hunk 1]
```diff
@@ -44,13 +34,13 @@ class Comment extends Entity
      * @param string $body
      * @param IClock $clock
      */
-    public function __construct(Post $post, string $body, IClock $clock)
+    public function __construct(Post $post, string $body)
     {
         parent::__construct();
         $this->post      = $post;
         $this->body      = $body;
-        $this->createdAt = new DateTime($clock->utcNow());
```
Comment (harikt) [inline provider/dms/Domain/Entities/Comment.php:52]: For the audience if we were using `datetime` field on the db, we could have used this. But as it is currently `bigint` , I am using time() ;) .

**Comment by sergeyklay** [comment]
@harikt Could you please rebase

File: provider/dms/Domain/Entities/Comment.php (hunk 1 above)
Comment (sergeyklay) [reply provider/dms/Domain/Entities/Comment.php:52]: Yes, I used int to simplify tests and avoid use yet another object to fill this field.

**Comment by harikt** [comment]
I have fixed the conflicts.

File: config/dms.php [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<?php
+
+return [
+    'dsn' => 'mysql://enigma:secret@mysql/orm_benchmark',
```
Comment (sergeyklay) [inline config/dms.php]: Not this should be 
```
mysql://enigma:secret@127.0.0.1/orm_benchmark
```

File: README.md [hunk 3]
```diff
@@ -17,6 +17,7 @@ NOTE: Some ORMs rely (depends) on models metadata caching. Thus, to avoid [contr
 * Phalcon 3.3.0
 * Propel ORM 2.0.0-alpha7
 * Yii ActiveRecord 2.0.13.1
+* DMS
```
Comment (sergeyklay) [inline README.md]: Could you add version here

**Review by sergeyklay — CHANGES_REQUESTED** [review]

**Comment by harikt** [comment]
Done 👍  .

**Comment by harikt** [comment]
dms currently fails for 7.2 for it has not released a version after it being patched. You can currently merge this.

**Review by sergeyklay — APPROVED** [review]

**Comment by sergeyklay** [comment]
@harikt Oh.. This branch cannot be rebased automatically via GitHub due to conflicts 😕  
screenshot 2018-01-10 09 29 43
Could you please rebase?

**Comment by harikt** [comment]
Do you really need a `rebase` or fetch upstream and merge ? I am not sure what are the steps I should exactly do here.

**Comment by harikt** [comment]
Can you click on command line instructions and see what it shows ?

**Comment by sergeyklay** [comment]
Ok, just merged w/o rebase :)

**Pull request merged**